External/ChemDraw/CMakeLists.txt

```diff
@@ -95,6 +95,10 @@ endif()
       endif(RDK_INSTALL_STATIC_LIBS)
   endif()
 
+  if(TARGET ChemDraw_static)
```

Comment:
Thanks, I REALLY think rdkit_library should do this though

Reply:
Well, there isn't, but expat's CMakeLists file still attempts to install it: :sweat_smile: https://github.com/Glysade/chemdraw/blob/main/expatpp/CMakeLists.txt#L37-L46

Reply:
I don't think that cmake variable is ever used...